src/dinterpret.d

```diff
@@ -4909,6 +4909,33 @@ public:
             fd = (cast(VarExp)ecall).var.isFuncDeclaration();
             assert(fd);
 
+            if (fd.ident == Identifier.idPool("__ctfeWrite"))
+            {
+                foreach(arg;*e.arguments)
```

Comment:
Why are you walking through the arguments twice? You can place this check & error in the foreach below.